test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go

```diff
@@ -222,6 +222,7 @@ func getValidValuesTests(toolToTest string) []latencyTest {
 	//testCpus: for tests that expect a success output message, note that an even CPU number is needed, otherwise the test would fail with SMTAlignmentError
 
 	successRuntime := "30"
+        testSet = append(testSet, latencyTest{testDelay: "200", testRun: "true", testRuntime: successRuntime, testMaxLatency: guaranteedLatency, testCpus: "4", outputMsgs: []string{success}, toolToTest: toolToTes$
```

Comment:
the syntax looks a bit off at the end :  `$` instead of `})  `

Reply:
Fixed

Reply:
still off ,  swap `))` with `})`

Reply:
Fixed